Sources/SwiftRoaring/RoaringBitmap.swift

```diff
@@ -72,41 +76,47 @@ public class RoaringBitmap: Sequence, Equatable, CustomStringConvertible,
     /// caller is
     /// responsible for memory management.
     ///
-    public func intersection(_ x: RoaringBitmap) -> RoaringBitmap {
-        return RoaringBitmap(ptr: croaring.roaring_bitmap_and(self.ptr, x.ptr))
+    @inlinable @inline(__always)
+    public func intersection(_ x: RoaringBitmap) -> Self {
+        return Self(ptr: croaring.roaring_bitmap_and(self.ptr, x.ptr))
     }
     ///
     /// Computes the intersection between two bitmaps and returns new bitmap. The
     /// caller is
     /// responsible for memory management.
     ///
-    public static func &(left: RoaringBitmap, right: RoaringBitmap) -> RoaringBitmap {
-        return left.intersection(right)
+    @inlinable @inline(__always)
+    public static func &(lhs: RoaringBitmap, rhs: RoaringBitmap) -> Self {
+        Self(ptr: croaring.roaring_bitmap_and(lhs.ptr, rhs.ptr))
     }
 
     ///
     /// Inplace version modifies x1, x1 == x2 is allowed
     ///
+    @inlinable @inline(__always)
```

Comment:
There is a lot of `@inline(__always)` involved here... I don't mind the `@inlinable`, though I am not sure it helps, but the `@inline(__always)` all over the code seems like abuse. I would only use it for some functions... like contains and so forth.

Reply:
I think it makes sense for all functions that just invoke the C functions.  As for using both, it's what swift collections uses for simple functions

 https://github.com/apple/swift-collections/blob/main/Sources/DequeModule/Deque%2BCollection.swift#L225